docs/src/lib/ClimaCoreTempestRemap.md

```diff
@@ -4,12 +4,32 @@
 CurrentModule = ClimaCoreTempestRemap
```

Comment:
What does `CurrentModule` refer to in this document? 
What would be the workflow if one wanted to test/use this to output lat-long grids? Can we document it?

Reply:
It's for [Documenter.jl](https://juliadocs.github.io/Documenter.jl/stable/man/guide/#Adding-Some-Docstrings)

Reply:
Thanks Charlie!